.github/workflows/ci.yaml

```diff
@@ -46,15 +44,9 @@ jobs:
       - name: Install solidity
         run: yarn global add solc@0.8.13
 
-      - name: "Create env file"
-        run: |
-          touch .env
-          echo FORK_MAINNET=${{ secrets.FORK_MAINNET }} >> .env
-          echo QUICK_NODE_KEY=${{ secrets.QUICK_NODE_KEY }} >> .env
```

Comment:
Here also we should keep the set of the env in theyam file and just pass `FORK_MAINNET` as false in github secrets to avoid running mainnet tests.

Reply:
We don't need this anymore because we have a test env file setup that is being sourced in the following step